unity/Assets/0Refactored/scenes/experiments/TitrationExperiment/scripts/AcidTitration.cs

```diff
@@ -0,0 +1,300 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System.IO;
+using System.Linq;
+using System;
+
+public class AcidTitration : MonoBehaviour
+{
+
+	Dictionary<string, int> strongBases = new Dictionary<string, int>() {

```

Comment:
Make fields readonly.
I think we can find a better way to define these things. But for now it's fine for me ;)